### DISCUSSION TITLE
Is there any way to save the issue record and mentioned record in my account after change my @username?

### DISCUSSION THREAD

**Discussion by jasminesor** [community/community#57900] [New to GitHub] [answered]
Hi, everyone! I am a github beginner and have been using the web version of github. I don't have a git client installed on my computer.

I want to change my github username, for example "@previousname" to "@newname". However, I submitted the issue in someone else's warehouse and left the conversation process on the issue page at "https://github.com/Marigold/universal-portfolios/issues/100". 
![image](https://github.com/community/community/assets/38998200/f25535c0-f157-4601-a228-94fcdeefa9a9)

This dialogue record is very important to me. I would like to ask whether the dialogue record of this issue can still be accessed through the above URL after changing my username? 

Besides, after changing my username, can the issue I mentioned still be found in my "closed issue" record and "mentioned" record? (as follows)
![image](https://github.com/community/community/assets/38998200/ed8e2990-bcf2-4b2a-944b-df9509d520eb)
![image](https://github.com/community/community/assets/38998200/16d66c86-0668-44c2-a646-6309986d721b)

**Comment by airtower-luna** [comment, marked as answer]
The issue shouldn't change at all, other than showing your new username where the arrow points in the first picture. It belongs to your *account*, not the (current) username. Of course this only applies if you change the name of your account, if you were to create a new account it wouldn't transfer over. :slightly_smiling_face: 

See the documentation: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username

**Reply by jasminesor** [reply]
Glad to get your reply : ). If changing the name keeps my repository and issue record, it looks like I can try changing the username. 
I've actually set up another account with an alternate email to keep my repository. Just worry about the issue record and whether my @previoususername mentioned in the issue record will change into my newusername. 
I think i can try it first now, your answer gives me confidence.

**Reply by airtower-luna** [reply]
An `@name` in the text/comments *won't* change (so it won't work any more), the documentation explicitly says that.

**Reply by jasminesor** [reply]
Is this means: if i change my username from `@A` to `@B`, i still can visit the issue page by the original url. But my mentioned record is still `@A` in the issue page and will not change. If some click the `@A` in the issue page, he/she cannot find anything? Am I right *_*?

**Reply by airtower-luna** [reply]
The username where you marked it in the screenshot should change, and lead to your new account page. But if someone [*mentioned you* with `@username`](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#mentioning-people-and-teams) in the text (e.g. a comment) that won't link to your new name.

**Reply by jasminesor** [reply]
I got it! Thank you very much my friend!

**Comment by jasminesor** [comment]
I made it, this discussion will be closed, thanks a lot @airtower-luna ~

**Reply by airtower-luna** [reply]
Excellent, glad I could help! :smile_cat:
